edx_lint/tamper_evident.py

```diff
@@ -53,18 +53,16 @@ def validate(self):
         with open(self.filename, "r") as f:
             text = f.read()
 
-        start_last_line = text.rfind(u"\n", 0, -1)
+        start_last_line = text.rfind("\n", 0, -1)
         if start_last_line == -1:
             return False
 
         original_text = text[:start_last_line+1]
         last_line = text[start_last_line+1:]
 
-        expected_hash = hashlib.sha1(original_text).hexdigest().encode("ascii")
+        expected_hash = hashlib.sha1(original_text.encode("utf-8")).hexdigest()
```

Comment:
This doesn't look right to me: on Py 2, original_text will be bytes already, so using .encode will raise an exception if there are non-ASCII characters.  My bad: there are no tests of non-ascii characters.